pages/_en-US/ds-index/hardmod.md

```diff
@@ -11,4 +11,102 @@ Hardmodding is when you physically solder the motherboard of the Nintendo DSi to
 ![Original DS motherboard with touch-points labeled](https://media.discordapp.net/attachments/547986366357700620/736370094392999947/mobo_pinout.png)
 
 ### Nintendo DSi
-Follow the "Hardware NAND Mod Guide" and "Decrypting your NAND" of [this](https://gbatemp.net/threads/dsi-downgrading-the-complete-guide.393682/) guide.
+
+To hardmod a Nintendo DSi / DSi XL (LL) you will need:
+  - [Pencil tipped soldering iron](https://www.amazon.com/dp/B01N4571Q6)
+  - [Very fine wire](https://www.amazon.com/dp/B01MXGNTA4), 28AWG or smaller, preferably 30AWG+ 
+  - SD to microSD adapter
+  - An SD card reader able to read eMMC chips running in single data-line mode. [Here's a known working one](https://www.amazon.com/dp/B006T9B6R2)
+  - [Win32DiskImager](https://sourceforge.net/projects/win32diskimager/) and [HxD](https://mh-nexus.de/en/downloads.php?product=HxD20) if on Windows
+  - A valid NAND backup of the DSi you are hardmodding
+  - To be comfortable soldering to pads that are as small as 0.5mm in diameter
+
+#### DSi side A pinouts
+![DSi side A pinouts](/assets/images/dsi-hardmod/side_a.jpg)
+#### DSi side B pinouts
+![DSi side B pinouts](/assets/images/dsi-hardmod/side_b.png)
+#### DSi XL side B pinouts
+![DSi XL side B pinouts](/assets/images/dsi-hardmod/dsi_xl_side_b.png)
+
+- You will need to solder the points on the motherboard to the microSD card adapter
+  - CMD to pin 2
+  - GND to pins 3 and 6
+  - CLK to pin 5
+  - DAT0 to pin 7
+
+#### Soldered microSD adapter example
+![microSD example](/assets/images/dsi-hardmod/sd.jpg)
+
+- Insert the microSD card adapter into your computer
+  - **WARNING** - Windows will ask you to format it: **DO NOT** format it - this might cause irrepairable damage
+
+- You will first need to remove the NOCASH footer from the backup you are flashing to the DSi
+- This can be done by running the hiyaCFW helper application found [here](https://github.com/mondul/HiyaCFW-Helper/releases/latest)
+
+1. Download the version for the operating system you are using
+1. Run the script, selecting the NAND backup you wish to flash to your DSi
+1. Select the output path to be anywhere on your computer
+
+- Once the script completes, there should be a folder at the selected output path with a long name starting with `FW`
+  - Use the file inside of this folder as the NAND image you flash onto the DSi
```

Comment:
I just tried it myself and a couple things:
- Probably good to mention to switch to NAND mode, as that's not super obvious that you need to click that button
- For me it just outputs `08A1807916109128-no-footer.bin` in the path where the terminal is, so the home folder if you run it on macOS, does it ask for a path and make the FW... folder for you?

```suggestion
1. Download the version for the operating system you are using
1. Run the script, selecting the NAND backup you wish to flash to your DSi
1. Switch to NAND mode with the button to the left of the file path
1. Click `Start` to create the NAND backup without the no$gba footer

- Once the script completes, there should be a file with a long name ending in `-no-footer.bin` in the folder where your terminal is open
  - Use this file as the NAND image you flash onto the DSi
```